src/Image/Image.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { props, t } from '../utils';
+import getUrl from './getUrl';
+
+export type ImageProps = {
+  /** path to the image */
+  src: string,
+  /** used to resize the image to this width before downloading it */
+  width?: number,
+  /** used to resize the image to this height before downloading it */
+  height?: number,
+  /** can be a number from 1 to 100 or "auto". Use "auto" to let cloudinary decide the quality for you */
+  quality?: number | 'auto',
+  /** which format the image should be. Use "auto" to let cloudinary decide the format for you */
+  format?: 'jpeg' | 'png' | 'wdp' | 'gif' | 'auto',
+}
+
+export const Props = {
+  src: t.String,
+  width: t.maybe(t.Number),
+  height: t.maybe(t.Number),
+  quality: t.maybe(t.union([t.Number, t.enums.of(['auto'])])),
+  format: t.maybe(t.enums.of(['jpeg', 'png', 'wdp', 'gif', 'auto']))
+};
+
+/** A replacement for `<img>` to serve optimized images through a CDN in production */
+@props(Props, { strict: false })
```

Comment:
Since we're using `strict: false`, the component should be able to receive every prop you can pass to `<img>` too. 
I would add `& React.ImgHTMLAttributes<HTMLImageElement` to `ImageProps`

Reply:
Good catch, done!

Reply:
could you please fix exports too?

Reply:
Which one?

Reply:
see the general comment to this PR. you're not exporting `Props` and `ImageProps` from `index.ts`

Reply:
Sorry, I missed that line!